### PR TITLE
Android Chrome search suggestions bug

### PR DESCRIPTION
**Pull request by tomdoughty**
## Description
On Android Chrome you cannot go to a search results page by touching a header search suggestion.

This is because of the `accessible-autocomplete` package blocking this event. They offer an `onConfirm` callback which is called on touch, click or enter on a focused option.

Dev has been completed to add this callback to our `headerAutoComplete`

## Checklist

- [x] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Follows our [coding standards and style guide](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [x] CHANGELOG entry

### CODE REVIEW

**Comment by pflynny** [comment]
@AdamChrimes the cors issue with the suggestions api have been resolved, so this is ready to go now.

**Comment by AdamChrimes** [comment]
@pflynny thanks Paul, please could you rebase with `master` and add an entry into the `CHANGELOG` and I'l take a look at the rest

File: packages/components/header/headerAutoComplete.js [hunk 1]
```diff
@@ -31,8 +29,8 @@ const suggestion = (result) => {
 */
 const source = (query, populateResults) => {
   // Build URL for search endpoint
-  const maxResults = 10;
-  const fullUrl = `${searchApiUrl}?collection=nhs-meta&partial_query=${query}&sort=0&fmt=json++&profile=&show=${maxResults}&q=${query}&api-version=1`;
+  // const maxResults = 10;
```
Comment (AdamChrimes) [inline packages/components/header/headerAutoComplete.js:32]: Can this commented out variable be removed if it's no longer used?

Comment (pflynny) [reply packages/components/header/headerAutoComplete.js]: My bad, I should've deleted that! I've fixed this now.

**Review by AdamChrimes — APPROVED** [review]

**Pull request merged**